ImplicitConverterTest.simpleDateFormat fails in certain time zones

The test was run at `2025-12-19T09:25:44-06:00` in `America/Chicago`.

The failure is:

```
[ERROR] Tests run: 10, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 0.017 s <<< FAILURE! -- in io.smallrye.config.ImplicitConverterTest
[ERROR] io.smallrye.config.ImplicitConverterTest.simpleDateFormat -- Time elapsed: 0.010 s <<< FAILURE!
org.opentest4j.AssertionFailedError: expected: <1970-01-01> but was: <1969-12-31>
	at org.junit.jupiter.api.AssertionFailureBuilder.build(AssertionFailureBuilder.java:151)
	at org.junit.jupiter.api.AssertionFailureBuilder.buildAndThrow(AssertionFailureBuilder.java:132)
	at org.junit.jupiter.api.AssertEquals.failNotEqual(AssertEquals.java:197)
	at org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:182)
	at org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:177)
	at org.junit.jupiter.api.Assertions.assertEquals(Assertions.java:1145)
	at io.smallrye.config.ImplicitConverterTest.simpleDateFormat(ImplicitConverterTest.java:212)
	at java.base/java.lang.reflect.Method.invoke(Method.java:565)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:511)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1450)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:2019)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:187)
```

The problem appears to be that the epoch time stamp is in 1969 in many parts of the world, so the simple date format test should adjust accordingly.

--- a/implementation/src/test/java/io/smallrye/config/ImplicitConverterTest.java
+++ b/implementation/src/test/java/io/smallrye/config/ImplicitConverterTest.java
@@ -18,6 +18,7 @@ import java.time.Instant;
 import java.time.LocalDate;
 import java.util.Date;
 import java.util.Locale;
+import java.util.TimeZone;
 
 import org.eclipse.microprofile.config.spi.Converter;
 import org.junit.jupiter.api.Test;
@@ -208,7 +209,10 @@ class ImplicitConverterTest {
         SmallRyeConfig config = new SmallRyeConfigBuilder()
                 .withSources(config("sdf", "yyyy-MM-dd"))
                 .build();
+
         SimpleDateFormat sdf = config.getValue("sdf", SimpleDateFormat.class);
+        sdf.setTimeZone(TimeZone.getTimeZone("UTC"));
+
         assertEquals("1970-01-01", sdf.format(Date.from(Instant.EPOCH)));
     }
 

